<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Home</class>
 <widget class="QMainWindow" name="Home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Home</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonPlay">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: rgb(148, 6, 6);
	background-color: rgb(255, 240, 73);
	font: 20pt &quot;Source Code Pro Medium&quot;;
	border-radius:10px;
}

QPushButton:hover{
	
	background-color: rgb(223, 223, 111);
}

QPushButton:pressed{
	background-color: rgb(212, 212, 106);
}</string>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 127);
font: 60pt &quot;Caladea&quot;;
</string>
    </property>
    <property name="text">
     <string>ROULETTE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>711</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/background.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="roulette.qrc">:/images/background.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonExit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>310</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: rgb(148, 6, 6);
	background-color: rgb(255, 240, 73);
	font: 20pt &quot;Source Code Pro Medium&quot;;
	border-radius:10px;
}

QPushButton:hover{
	
	background-color: rgb(223, 223, 111);
}

QPushButton:pressed{
	background-color: rgb(212, 212, 106);
}</string>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonHowToPlay">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>210</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: rgb(148, 6, 6);
	background-color: rgb(255, 240, 73);
	font: 20pt &quot;Source Code Pro Medium&quot;;
	border-radius:10px;
}

QPushButton:hover{
	
	background-color: rgb(223, 223, 111);
}

QPushButton:pressed{
	background-color: rgb(212, 212, 106);
}</string>
    </property>
    <property name="text">
     <string>How to Play</string>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>pushButtonPlay</zorder>
   <zorder>label</zorder>
   <zorder>pushButtonExit</zorder>
   <zorder>pushButtonHowToPlay</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>707</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="roulette.qrc"/>
 </resources>
 <connections/>
</ui>
